### PR TITLE
Update docstrings to reflect new parameters

### LINKED ISSUE #70: update docstrings in importer
We added repo_id to a bunch of methods

https://github.com/mitodl/lore/pull/64/files#r31520091
https://github.com/mitodl/lore/pull/64/files#r31520120
https://github.com/mitodl/lore/pull/64/files#r31562349

Hopefully these links to the discussion will survive the PR being merged.

### PR DESCRIPTION
**Pull request by pwilkins**
- Update docstrings to reflect newly added parameters in Importer.
- Corrected docstrings that weren't rendering properly.
- Added Taxonomy app to Sphinx build

Closes #70

### CODE REVIEW

File: learningresources/api.py [hunk 1]
```diff
@@ -111,7 +111,7 @@ def get_repos(user_id):
     Args:
         user (auth.User): request.user
```
Comment (carsongee) [inline learningresources/api.py]: Can you fix this one? It says auth.User but is using the primary key

**Comment by pwilkins** [comment]
Please review - @pdpinch @noisecapella @carsongee 
The easiest way to review the rendered doc is by viewing it on ReadTheDocs: http://lore.readthedocs.org/en/feature-pw-importer-docstrings/

File: learningresources/api.py [hunk 2]
```diff
@@ -184,11 +191,12 @@ def get_user_tags(repo_id):
 
 def get_resources(repo_id):
     """
-    Get resources from a repository.
+    Get resources from a repository ordered by title.
+
     Args:
         repo_id (int): primary key of the repository
```
Comment (carsongee) [inline learningresources/api.py]: If we are upcasing the first word like above with the `primary key -> Primary key` changes, we should do that consistently like here and `list of resources`, and `resources` unless I am not grasping the reason for the changes above.

**Comment by carsongee** [comment]
:taxi:

**Pull request merged**